Post processing update for commercial measures

lyt5410 opened the pull request:
# Pull Request (PR) Description
**NOTE: SUBMIT A PLAN TO DEER SUPPORT PRIOR TO WORKING ON CODE FOR A MEASURE, FEATURE, OR BUG TO BE CONNECTED WITH OTHERS WORKING ON THE SAME ISSUE OR TO RECEIVE FEEDBACK ON YOUR PROPOSED METHOD **

**DEER Support Email: DeerSupport@dnv.com**

### PR Author: Yitian Liang
- [x] Craft a succinct title that effectively encapsulates the essence of the pull request, providing a general overview of the proposed changes.
- [x] Label the PR with at least one of the following: New Measure, Bug, or Feature.
- [x] Provide a concise description of the measure, bug, or feature. Submit one PR per measure.
**Description: 
this is an update that includes a set of python scripts (in the "data transformation" folder), SQL scripts (in the "energy savings" folder), and also supplementary scripts for pulling DEER weights from the DEER database. An example made-up measure "SWXX111-00 Example_SEER_AC" set up is provided and have the building types Asm, ECC, EPr enable for 1975 vintage, with the Com.py scripts working alongside this example setup/results for the post-processing workflow. 
Also, the scripts "DEER_weights_extraction.py" and "DEER_weights_extraction_res.py" is included to pull weights tables as csv, directly from the DEER database. The weights table are support table used in the SQL scripts for calculating measure impacts.**


- [ ] For a new measure, attach a workbook named DEER_EnergyPlus_Modelkit_Measure_list_working.xlsx, containing only rows used for post-processing the measure.
- [ ] Add comments in the code when necessary to facilitate the review process.
- [ ] Add a comment before the added code, including the author's full name, company, and specifying if it's a bug fix, new measure, or feature.
- [x] For a new feature or bug, demonstrate the impact on energy consumption for selected cases with justification using plots and descriptions.
"SWXX111-00 Example_SEER_AC"  in the "commercial measures" folder is an example made-up measure created to show an example workflow with the Com.py script and all the subsequent steps.

- [ ] For a new measure, add a summary table showing total energy consumption per simulated case.
### PR Reviewer
- [ ] Conduct a thorough code review.
- [ ] If the branch is behind the main, merge the branch locally to check for potential conflicts.
- [ ] If a bug, locally reproduce it and compare energy consumptions before and after.
- [ ] Explore creative ways to stress-test the code.
- [ ] Locally check the error file and other outputs.

amine-lazrak commented:
Please review and update following:

- [ ] Update the [example folder](https://github.com/sound-data/DEER-Prototypes-EnergyPlus/tree/post-processing-update-Com/commercial%20measures/SWXX111-00%20Example_SEER_AC) to be specific to Com post processing.
- [ ] Update [Readme ](https://github.com/sound-data/DEER-Prototypes-EnergyPlus/blob/post-processing-update-Com/scripts/Readme.md)to explain the post processing process.
- [ ] Clean cases files (remove instead of comment).
- [ ] Make sure you pull the most recent changes from main before you submit the PR.

lyt5410 commented:
> Please review and update following:
> 
> * [ ]  Update the [example folder](https://github.com/sound-data/DEER-Prototypes-EnergyPlus/tree/post-processing-update-Com/commercial%20measures/SWXX111-00%20Example_SEER_AC) to be specific to Com post processing.
Added a comment on line 26 of Com.py
> * [ ]  Update [Readme ](https://github.com/sound-data/DEER-Prototypes-EnergyPlus/blob/post-processing-update-Com/scripts/Readme.md)to explain the post processing process.
> * [ ]  Clean cases files (remove instead of comment).
Cleaned.
> * [ ]  Make sure you pull the most recent changes from main before you submit the PR.
Rebased current branch before making above changes
